config/dev.exs

```diff
@@ -57,7 +57,7 @@ config :animina, AniminaWeb.Endpoint,
 config :animina, AniminaWeb.Endpoint,
   live_reload: [
     patterns: [
-      ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
+      ~r"priv/static/(?!uploads/).*(js|css|png|jpeg|jpg|gif|svg)$",
```

Comment:
In my setup I get an error here because I don't have a `uploads` directory. Please check it into the repo with a `.gitkeep` file in it.